3_Automation/06_Bestiary.yml

```diff
@@ -0,0 +1,359 @@
+      AOE: (aura, cone, line) X       # line would be X by Y, requires new diagram
+      Target: X                       # number: 1 for single, 2 for 2 targets
+      Effect: Descrip
+      Save:
+        Trigger: Once, On hit, Start turn, etc
+        DR: X
+        Type: MOD                     # AGL, CON, etc
+        Fail: Descrip                 # effect on fail
+        Succeed: Descrip              # effect on success. Assume no access to above
+  Threat Level: X                     # Consistent comparable rating fitting formula
+  Phases:                             # only for Boss
+    One:
+      HP: X
+      Allies:
+  Element: Earth, Fire, etc           # Setting-specific
+  Description: Descrip                # Full descriptive text
+
+
+  Falcon:
+  Type: Companion, level 1
+  HP: 1
+  Passives:
+    Keen Eyed: When their Dealer makes a Detection check as an action with the companion present, they do so with Upper Hand
+  Action:
+    Swoop:
+      Type: Minor
+      Target: 1
+      Effect: Next attack against the target has Upper Hand
+  Element: Air
+
+Bear:
+  Type: Companion, level 1
+  HP: 3
+  Actions:
+    Bear Hug: 
+      Type: Minor
+      Range: 0
+      Target: 1
+      Effect: Grapples target enemy
+      Save:
+        Trigger: Once
+        DR: 2
+        Type: STR 
+        Fail: All attacks against the target made until the end of their next turn have Upper Hand.
+  Element: Earth
+
+Wolf:
+  Type: Companion, level 1
+  HP: 2
+  Passive:
+    Mount: When ridden, grants +3 to rider.
+  Actions:
+    Pack attack:
+      Type: Minor
+      Range: 0
+      Effect: If they are next to at least 2 enemies, all attacks against the target until the end of their turn have Upper Hand.
+  Element: Fire
+
+Horse:
+  Type: Companion, level 1
+  HP: 2
+  Passive:
+    Mount: When ridden, grants +3 to rider.
+    Elevated: Melee attacks against the rider are made at Lower Hand
+  Element: Water
+
+Wolf:
+  Type: Bystander
+  HP: 2
+  AP: 0
+  AR: 3
+  Skill Mods: 
+    AGL: 1
+    STR: 1
+  Actions:
+    Pack attack:
+      Type: Minor
+      Range: 0
+      Effect: If they are next to at least 2 enemies, all attacks against the target until the end of their turn have Upper Hand.
+    Attack, Weapon:
+      Type: Major
+      To hit: 1
+      Damage: 1            
+      Range: 0
+      Target: 1       
+  Primary Element: Earth
+
+Dire Wolf:
+  Type: Bystander
+  HP: 3
+  AP: 0
+  AR: 2
+  Skill Mods: 
+    AGL: 2
+    STR: 2
+  Actions:
+    Pack attack:
+      Type: Minor
+      Range: 0
+      Effect: If they are next to at least 2 enemies, all attacks against the target until the end of their turn have Upper Hand.
+    Pounce:                             
+      Type: Major
+      To hit: 2
+      Damage: 2             
+      Range: 0
+      Target: 1                 
+      Effect: Pouncing on the enemy, you make an attack against a creature next to you and attempt to knock them to the ground.
+      Save:
```

Comment:
Should this (or any subset of Beastiary powers) be available to PCs?

Reply:
We'll streamline what is/isn't accessible to PCs in the Powers List future commits.